packages/@nucleus/tests/dummy/app/components/theme-toggle.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@ember/component';
+import { set } from '@ember/object';
+
+export default Component.extend({
+  dark: true,
+
+  toggleDarkTheme(match) {
+    document.body.classList.toggle('dark', match);
+  },
+  init() {
+    this._super();
+    const prefersDark = window.matchMedia('(prefers-color-scheme: dark)');
+    if (prefersDark.matches) {
+      set(this, 'dark', true);
+    }
+    else {
+      set(this, 'dark', false);
```

Comment:
`set(this, 'dark', prefersDark.matches)` ?

Reply:
Sure! 😅